config/locales/loa1/en.yml

```diff
@@ -0,0 +1,27 @@
+---
+en:
+  loa:
+    cancel:
+      modal_header: Are you sure you want to cancel?
+      warning_header: If you cancel now
+  loa1:
+    buttons:
+      continue: Continue account creation
+      cancel: Cancel and delete your information
+    cancel:
+      success: The record of your information has been cleared
+      warning_points:
+        - You won’t have a login.gov account
+        - We won't keep a record of your email address, password, and phone number
+        - You won't be able to access your benefits information with the Railroad Retirement Board
```

Comment:
Modal will theoretically apply to any service provider who needs to handle sign ups or idv via login.gov. I'm not sure how it would be re-written to not be redundant with the bullet points that have already been enumerated. It seems obvious that they wouldn't be able to see details about an account that doesn't exist.

My thinking was that the information a service provider passes along to login.gov would include some kind of message around the sign up benefits, but it seemed out of scope for this ticket